File: HealthAxis.Api/Models/Patient.cs
using System.ComponentModel.DataAnnotations;

namespace HealthAxis.Api.Models
{
    public class Patient
    {
        [Key]
        public int PatientId { get; set; }
        [Required]
        [RegularExpression(@"[A-Z][A-za-z\s]+", ErrorMessage = "Name should contain only Alphabets")]
        [MinLength(2)]
        public required string FullName { get; set; }
        [Required]
        public DateTime DateOfBirth { get; set; }
        [Required]
        [RegularExpression("(Male|Female|Transgender|Other)", ErrorMessage = "Gender should be Male , Female , Transgender or other")]
        public required string Gender { get; set; }
        [Required]
        [EmailAddress(ErrorMessage = "Email is invalid")]
        public required string Email { get; set; }
        [Required]
        [Phone(ErrorMessage = "Phone Number is invalid")]
        public required string PhoneNumber { get; set; }
        [RegularExpression(@"^$|^INS\d{4}$", ErrorMessage = "Format must be INSXXXX (4 digits)")]
        public string? InsuranceID { get; set; }
        public bool IsActive { get; set; }
    }
}

File: HealthAxis.Angular/src/app/dtos/patient.ts
export interface PatientProfile {

  patientId: number;

  fullName: string;

  dateOfBirth: string;

  gender: string;

  email: string;

  phoneNumber: string;

  insuranceID?: string;

  isActive: boolean;
}

export interface CreatePatientProfileRequest {

  fullName: string;

  dateOfBirth: string;

  gender: string;

  email: string;

  phoneNumber: string;

  insuranceID?: string;
}

export interface UpdatePatientProfileRequest {

  patientId: number;

  fullName: string;

  dateOfBirth: string;

  gender: string;

  phoneNumber: string;

  insuranceID?: string;

  isActive: boolean;
}